### PR TITLE
Add SMTP Credential Hashes spec

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: spec/paths/credential-hashes@email@{token}.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+parameters:
+  - $ref: "#/parameters/token"
+get:
+  tags:
+    - Credential Hashes
```
Comment (slavcodev) [inline spec/paths/credential-hashes@email@{token}.yaml]: The different tag was added in tags list

File: spec/definitions/EmailCredentials/SmtpAuthorization.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+type: object
+discriminator: type
+properties:
+  type:
+    type: string
+    enum:
+      - none
```
Comment (slavcodev) [inline spec/definitions/EmailCredentials/SmtpAuthorization.yaml]: Could we do this default?

Comment (vova07) [inline spec/definitions/EmailCredentials/SmtpAuthorization.yaml]: I think only @adamaltman can answer to this question.

Comment (slavcodev) [inline spec/definitions/EmailCredentials/SmtpAuthorization.yaml]: OK, but for `encryption` seems you've decided yet :)

**Comment by vova07** [comment]
Actually I didn't. It was like this in old code, so I just copied the logic. But I think you are right. Waiting for Adam's reply.

File: spec/definitions/EmailCredentials/SmtpAuthorization.yaml (hunk 2 above)
Comment (adamaltman) [inline spec/definitions/EmailCredentials/SmtpAuthorization.yaml]: Yes, it could be default.

File: spec/swagger.yaml [hunk 3]
```diff
@@ -222,6 +222,9 @@ tags:
       website is related to each invoice and each payment gateway account.  This feature
       would allow you to have gateway accounts that are related to multiple websites, or
       exclusive to particular websites.  And gives you more control over your business.
+  - name: Credential Hashes
```
Comment (adamaltman) [inline spec/swagger.yaml]: Please put it alphabetical.

File: spec/swagger.yaml [hunk 4]
```diff
@@ -319,6 +322,12 @@ parameters:
     in: query
     type: string
     description: Expand response to get full related object intead of ID.  See the expand guide for more info.
+  token:
```
Comment (adamaltman) [inline spec/swagger.yaml]: why is this here?

Comment (slavcodev) [inline spec/swagger.yaml]: It's reusable request parameter https://github.com/Rebilly/RebillyAPI/pull/126/files#diff-8a487e16a43f859c7eb558d90991d53bR2

File: spec/definitions/EmailCredentials/SmtpAuthorization.yaml (hunk 2 above)
Comment (vova07) [inline spec/definitions/EmailCredentials/SmtpAuthorization.yaml]: done!

File: spec/swagger.yaml (hunk 3 above)
Comment (vova07) [inline spec/swagger.yaml]: done!

File: spec/paths/credential-hashes@email@{token}.yaml (hunk 1 above)
Comment (vova07) [inline spec/paths/credential-hashes@email@{token}.yaml]: Fixed!

File: spec/paths/credential-hashes@emails@{token}.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+parameters:
+  - $ref: "#/parameters/token"
```
Comment (slavcodev) [inline spec/paths/credential-hashes@emails@{token}.yaml]: Is not that called `hash` now?

Comment (vova07) [inline spec/paths/credential-hashes@emails@{token}.yaml]: yes. Thanks! Fixed!
